pages/api/cep/v2/[cep].js

```diff
@@ -23,6 +23,11 @@ async function Cep(request, response) {
       cepFromCepPromise
     );
 
+    if (!cepFromCepPromise.street) {
+      cepFromCepPromise.neighborhood = null;
+      cepFromCepPromise.street = null;
+    }
```

Comment:
único detalhe é que eu separaria 
```suggestion
    if (!cepFromCepPromise.street) {
      cepFromCepPromise.street = null;
    }
    
    if (!cepFromCepPromise.neighborhood) {
      cepFromCepPromise.neighborhood = null;
    }
```

Reply:
Pensei nisso, mas será que faz sentido? Porque se for CEP único, `street` e `neighborhood` vão vir como `null`, mas se não for CEP único, `street` vai ter um valor, por isso acho que só precisa testar se `street` é `null`. O que você acha?

Reply:
Opa, @jvolima!
Não manjo muito de cep, mas fui consultar um camarada que já fez alguns trabalhos para agências do correio, ele me disse que existem ceps únicos por bairro tbm, ou seja, podemos ter casos onde não tem o endereço, mas tem o bairro.

Ele me mandou uma documentação interessante sobre ceps: https://www.correios.com.br/enviar/precisa-de-ajuda/tudo-sobre-cep

Então acredito que a sugestão do @LorhanSohaky faça sentido.

Reply:
Entendi @RodriAndreotti, confesso que não fazia ideia que existia CEP único por bairro. Vou fazer o push das modificações, valeu!

Reply:
Pois é, também só descobri depois que li o comentário do Lorhan... rs